<launch>
  <arg name="pcd_file" default="20190729_good_design_1.pcd" />
  <node pkg="pev_map_server" type="pcd_to_pgm_map" name="pcd_to_pgm_map_node" output="screen">
    <!-- 0: off visualization, 1: all cloud, 2: obstacel cloud, 3: ground, 4: negtive ground -->
    <param name="visualization" type="int" value="2" />
    <param name="min_z" type="double" value="0.5" />
    <param name="max_z" type="double" value="10.9" />
    <param name="seg_distance_th" type="double" value="0.25" />
    <param name="pcd_file" type="str" value="$(env HOME)/pcd/$(arg pcd_file)" />
  </node>
</launch>
